pkg/ui/src/views/shared/components/sortabletable/index.tsx

```diff
@@ -128,16 +155,49 @@ export class SortableTable extends React.Component<TableProps, {}> {
             "sort-table__row",
             "sort-table__row--body",
             this.props.rowClass(rowIndex),
+            { "sort-table__row--expandable": !!expandableConfig },
           );
-          return (
-            <tr key={rowIndex} className={classes}>
+          const expanded = expandableConfig && expandableConfig.rowIsExpanded(rowIndex);
+          const output = [
+            <tr
+              key={rowIndex}
+              className={classes}
+              onClick={() => expandableConfig.onChangeExpansion(rowIndex, !expanded)}
```

Comment:
Seems like this should be defensive to null.

Reply:
Done